ui/components/app/app-header/app-header.component.js

```diff
@@ -152,6 +159,10 @@ export default class AppHeader extends PureComponent {
                 }
                 history.push(DEFAULT_ROUTE);
               }}
+              ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+              custodyImgSrc={custodianIcon}
```

Comment:
Can we have 1 snapshot/dom snap insertion to cover this element in `ui/components/app/app-header/app-header.test.js`? :)

Reply:
Added tests in MetaFox logo instead of app-header